### PR TITLE
add test in descibe resourcequota

### PR DESCRIPTION
**Pull request by zjj2wry**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by rootfs** [comment]
@k8s-bot ok to test

File: pkg/printers/internalversion/describe_test.go [hunk 1]
```diff
@@ -1422,3 +1422,42 @@ URL:	http://localhost
...
 }
+
+func TestDescribeResourceQuota(t *testing.T) {
+	fake := fake.NewSimpleClientset(&api.ResourceQuota{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "bar",
+			Namespace: "foo",
+		},
+		Status: api.ResourceQuotaStatus{
+			Hard: api.ResourceList{
```
Comment (rootfs) [inline pkg/printers/internalversion/describe_test.go]: It is odd to see request CPU/memory equal to resource CPU/memory

File: pkg/printers/internalversion/describe_test.go [hunk 2]
```diff
@@ -1422,3 +1422,42 @@ URL:	http://localhost
...
+			},
+		},
+	})
+	c := &describeClient{T: t, Namespace: "foo", Interface: fake}
+	d := ResourceQuotaDescriber{c}
+	out, err := d.Describe("foo", "bar", printers.DescriberSettings{ShowEvents: true})
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	expectedOut := []string{"bar", "foo", "limits.cpu", "1", "requests.memory", "4G"}
```
Comment (rootfs) [inline pkg/printers/internalversion/describe_test.go]: also test request cpu and memory

**Comment by zjj2wry** [comment]
@k8s-bot pull-kubernetes-unit test this

**Comment by zjj2wry** [comment]
@k8s-bot pull-kubernetes-e2e-gce-etcd3 test this

File: pkg/printers/internalversion/describe_test.go (hunk 1 above)
Comment (zjj2wry) [inline pkg/printers/internalversion/describe_test.go]: i updated it. this metrics is my test namespace quota.

**Comment by zjj2wry** [comment]
@rootfs review it again. think you~

**Pull request merged**